src/renderer/components/App.tsx

```diff
@@ -18,23 +17,13 @@ import * as dialogActions from "readium-desktop/common/redux/actions/dialog";
 import * as styles from "readium-desktop/renderer/assets/styles/app.css";
 import DialogManager from "readium-desktop/renderer/components/dialog/DialogManager";
 import PageManager from "readium-desktop/renderer/components/PageManager";
-import { lazyInject } from "readium-desktop/renderer/di";
-import { RootState } from "readium-desktop/renderer/redux/states";
-import { Store } from "redux";
+import { diRendererGet } from "readium-desktop/renderer/di";
 
-import { diRendererSymbolTable } from "../diSymbolTable";
-import { IRouterLocationState } from "../routing";
 import DownloadsPanel from "./DownloadsPanel";
 import ToastManager from "./toast/ToastManager";
 
 export default class App extends React.Component<{}, undefined> {
 
-    @lazyInject(diRendererSymbolTable.store)
-    private store: Store<RootState>;
-
-    @lazyInject(diRendererSymbolTable.history)
-    private history: History<IRouterLocationState>;
```

Comment:
This change is interesting. Are you recommending removing *all* occurrences of `@lazyInject` in the renderer process code? (and instead replace with `diRendererGet("xx")` wherever required?) Do you think `@lazyInject` is unreliable, or is this change motivated by personal coding taste?

Reply:
`lazyInject` causes problems when importing from redux-saga

Reply:
Ok, so this PR removes all usages of `@lazyInject`.